src/matchers.rs

```diff
@@ -9,6 +9,7 @@
 //! Check [`Match`]'s documentation for examples.
 use crate::{Match, Request};
 use assert_json_diff::{assert_json_matches_no_panic, CompareMode};
+use base64::prelude::*;
```

Comment:
Let's avoid star imports please 🙏

Reply:
IMHO star imports are allowed for prelude, but anyway I'll fix that.